server/README.md

```diff
@@ -0,0 +1,19 @@
+## 🔄 **Regenerating `package.json` and `package-lock.json`**
+
+If you've accidentally deleted the `package.json` or `package-lock.json` files in the **server** directory, you can follow these steps to regenerate them.
+
+### 1️⃣ **Regenerate `package.json`**
+
+To regenerate the `package.json` file with default values, run the following command in the **server** directory:
+
+```bash
+npm init -y
```

Comment:
this is not enough as it just generates an empty `package.json` file with the following contents
```bash
npm init -y
Wrote to /home/q550356/sample_code/server/package.json:

{
  "name": "server",
  "version": "1.0.0",
  "description": "If you've accidentally deleted the `package.json` or `package-lock.json` files in the **server** directory, you can follow these steps to regenerate them.",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}

```

Reply:
I tried using the previous package file from https://github.com/fsheikh/sample_code/blob/deurdu/deurdu/server/package.json but then had to `npm install express` and `npm install sqlite3`

Reply:
Oh ok, by the way I did that thing: `npm init -y` and it generated me that package.json file.
I don't know why, but we will discuss it on call.